react/components/EXPERIMENTAL_Charts/helpers.ts

```diff
@@ -13,7 +13,7 @@ const merge = (defaultProps: any, userProps: any) => {
   return props
 }
 
-const getRangeOfZAxis = (key, data) => {
+const getRangeOfZAxis = (key: string | number, data: unknown[]) => {
```

Comment:
```suggestion
const getRangeOfZAxis = (key: string | number, data: object[]) => {
```
To be consistent with `typings.d.ts`?

Reply:
Sure